src/sentry/api/endpoints/team_projects.py

```diff
@@ -16,7 +21,7 @@
 ERR_INVALID_STATS_PERIOD = "Invalid stats_period. Valid choices are '', '24h', '14d', and '30d'"
 
 
-class ProjectSerializer(serializers.Serializer):
+class ProjectPostSerializer(serializers.Serializer):
```

Comment:
We're required to change the name of this serializer b/c there is a duplicate `ProjectSerializer` in the projects model and drf-spectacular/openapi yells at me